src/components/summary-list/index.md.njk

```diff
@@ -30,24 +30,31 @@ Do not use it for tabular data or a simple list of information or tasks, like a
 
 There are 2 ways to use the summary list component. You can use HTML or, if you’re using [Nunjucks](https://mozilla.github.io/nunjucks/) or the [GOV.UK Prototype Kit](https://govuk-prototype-kit.herokuapp.com), you can use the Nunjucks macro.
 
-### Summary list with actions
+{{ example({group: "components", item: "summary-list", example: "without-actions", html: true, nunjucks: true, open: false}) }}
+
+###  Adding actions to each row
 
 You can add actions to a summary list, like a ‘Change’ link to let users go back and edit their answer.
 
 For sighted users, the actions get their context from the other content in the row they appear in.
 
 Assistive technology users, like those who use a screen reader, may hear the links out of context and not know what they do.
+
 To give more context, add visually hidden text to the links. This means a screen reader user will hear a meaningful action, like ‘Change name’ or ‘Change date of birth’.
 
 {{ example({group: "components", item: "summary-list", example: "default", html: true, nunjucks: true, open: false, titleSuffix: "second"}) }}
 
-### Summary list without actions
+### Adding actions to only some rows
 
-{{ example({group: "components", item: "summary-list", example: "without-actions", html: true, nunjucks: true, open: false}) }}
+If you have a mix of rows with and without actions, add the `govuk-summary-list__row--no-actions` modifier class to the rows that do not have actions.
+
+{{ example({group: "components", item: "summary-list", example: "mixed-actions", html: true, nunjucks: true, open: false}) }}
+
+### Removing the borders
 
-### Summary list without borders
+The summary list includes separating borders to help users read each row and its action.
```

Comment:
I know adding this explanation is a bit clunky, but it looked like adding a "do not do this" line afterwards started to look a bit confusing (and easy to miss if it went after the example).